Aptoide/aptoide-client-v8
# Feature/ASV-175-AppsFragment

**What does this PR do?**
The goal of this pull request was to merge the views form the updates tab and the downloads tab.  Now the updates progress should not be on the downloads section, but it should be on the own updates section. 
There is a section called Apps on the bottom navigation that compiles the downloads, updates and installed apps.

**Database changed?**

   No 

**Where should the reviewer start?**

- AppsFragment
- AppsPresenter
- AppsManager

**How should this be manually tested?**

[DOWNLOAD] - Open an appview click install , change to the Apps on the bottom navigation and see the progress. When the download is finished, a popup to install the app should be shown. If the app is installed then the card should be removed.

[UPDATE] - Open Apps on the bottom navigation, start an update and see the progress. When the update is finished, then a popup to install the app  (new version) should be shown.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-175](https://aptoide.atlassian.net/browse/ASV-175)

**Questions:**
N/A

**Code Review Checklist**
